### PR TITLE
Remove default for filed by va gov on the HLR

### LINKED ISSUE #1935: BVA Dispatch Manager view - update column header
BVA Dispatch Manager view - change 'Days in Queue' column header to 'Days Since Outcoding'.

When a claim is outcoded it is not always uploaded at the same time. It is not technically in the Dispatch queue until it has been uploaded to VBMS, this is sometimes days later.

### PR DESCRIPTION
**Pull request by nweinmeister**
Resolves #1935

### Description
For HLRs we don't want the filed_by_va_gov field to default to No and it needs to be required since the HLR efiling is already live.

This also moves the logic for the HLR feature toggle check from the FormGenerator to the review page which I think makes more sense since the FormGenerator shouldn't have too much knowledge of the specific forms functionality.

### CODE REVIEW

File: client/app/intake/pages/formGenerator.jsx [hunk 1]
```diff
@@ -52,6 +52,17 @@ const rampElectionRadioOptions = map(rampElectionReviewOptions, (option) => ({
...
 const formFieldMapping = (props) => {
+  const isAppeal = props.formName === FORM_TYPES.APPEAL.formName;
+  const renderVaGovValue = () => {
+    // eslint-disable-next-line no-undefined
+    if (isAppeal && (props.filedByVaGov === null || props.filedByVaGov === undefined)) {
+      return 'false';
+    }
+
+    // eslint-disable-next-line no-undefined
+    return props.sameOffice === null || props.sameOffice === undefined ? null : props.filedByVaGov.toString();
```
Comment (nanotone) [inline client/app/intake/pages/formGenerator.jsx:63]: Why does the VaGov logic here depend on sameOffice?

Comment (nweinmeister) [reply client/app/intake/pages/formGenerator.jsx]: Forgot to push up the fix for this but it's updated now

**Review by nanotone — APPROVED** [review]
I had just the one extra suggestion about that function, but whether it improves or whether you think it's not worth it, this is good to go.

File: client/app/intake/pages/formGenerator.jsx [hunk 2]
```diff
@@ -52,6 +52,17 @@ const rampElectionRadioOptions = map(rampElectionReviewOptions, (option) => ({
...
 const formFieldMapping = (props) => {
+  const isAppeal = props.formName === FORM_TYPES.APPEAL.formName;
+  const renderVaGovValue = () => {
+    // eslint-disable-next-line no-undefined
+    if (isAppeal && (props.filedByVaGov === null || props.filedByVaGov === undefined)) {
+      return 'false';
+    }
+
+    // eslint-disable-next-line no-undefined
+    return props.filedByVaGov === null || props.filedByVaGov === undefined ? null : props.filedByVaGov.toString();
```
[the remark this answers is not in the document]
Comment (meckila-navapbc) [reply client/app/intake/pages/formGenerator.jsx]: +1️⃣

[the remark this answers is not in the document]
Comment (nweinmeister) [reply client/app/intake/pages/formGenerator.jsx]: Did a slightly different approach and pulled all the bool value logic into a shared method to use

**Review by meckila-navapbc — APPROVED** [review]

**Pull request merged**